test/sql/storage/compression/bitpacking/bitpacking_fatal_forced.test_coverage

```diff
@@ -0,0 +1,25 @@
+# name: test/sql/storage/compression/bitpacking/bitpacking_fatal_forced.test_coverage
+# description: Test forced bitpacking, with value ranges that are rejected by the bitpacking compression analyze step
+# group: [bitpacking]
+
+# load the DB from disk
+load __TEST_DIR__/test_bitpacking.db
+
+statement ok
+PRAGMA force_compression = 'bitpacking'
+
+statement ok
+CREATE TABLE test (a INT64, b INT32, c INT16, d INT8);
+
+statement ok
+INSERT INTO test SELECT i, i, i, i FROM range(-128, 128) tbl(i);
```

Comment:
I have made a change to `force_compression`'s behavior so it falls back to UNCOMPRESSED instead of failing altogether.

As a result of this I am checking the compression used with `storage_info` for each column separately to confirm the correct compression method is used.

Would you still like me to make more tests for this or does that meet the requirements of your request?

Reply:
The tests you added there lgtm!